pkg/oc/cli/registry/login/login.go

```diff
@@ -174,10 +175,10 @@ func (o *LoginOptions) Complete(f kcmdutil.Factory, args []string) error {
 			return fmt.Errorf("the service account %s had no valid secrets associated with it", o.ServiceAccount)
 		}
 	default:
-		if len(cfg.BearerToken) == 0 {
+		if cfg.WrapTransport == nil {
```

Comment:
@smarterclayton ptal at this change, this was what was causing the problems. https://github.com/kubernetes/kubernetes/pull/67359 introduced a periodical reading of token file for inClusterConfig and introduced this field.